controllers/start.go

```diff
@@ -87,13 +89,27 @@ func StartManager(namespace string, metricsAddr string, enableLeaderElection boo
 		defer publisher.Close()
 	}
 
+	// Create a globally-scoped Kubernetes client. mgr.GetClient() can only
+	// refer to resources in the same-namespace when one is provided.
+	cfg, err := rest.InClusterConfig()
```

Comment:
I would just add a line above this to check if out-of-cluster is available for development purposes. If that fails, then it can cascade down to in-cluster and bomb if that fails